mypy.ini

```diff
@@ -596,6 +596,21 @@ ignore_errors = True
 [mypy-scipy.ndimage.morphology]
 ignore_errors = True
 
+[mypy-scipy.ndimage._filters]
+ignore_errors = True
+
+[mypy-scipy.ndimage._fourier]
+ignore_errors = True
+
+[mypy-scipy.ndimage._interpolation]
+ignore_errors = True
+
+[mypy-scipy.ndimage._measurements]
+ignore_errors = True
+
+[mypy-scipy.ndimage._morphology]
```

Comment:
@rgommers I've added underscores here but did not remove the old ones yet. I guess it is okay to get rid of
LOC 584 to 597 (GitHub is not letting me select all these lines to comment for some reason.)